playwright-e2e/dsm/pages/participant-list-page.ts

```diff
@@ -264,7 +279,8 @@ export default class ParticipantListPage extends DsmPageBase {
           }
           unformattedFirstName = JSON.stringify(value.esData.profile.firstName);
           firstName = unformattedFirstName.replace(/['"]+/g, ''); //Replace double quotes from JSON.stringify
-          if (medicalRecord && (firstName === testParticipantFirstName)) {
+          const participantID = value.participant.participantId; //Make sure when searching for onc history that the participant has a participantId in ES
```

Comment:
not sure if 'participantId in ES' is the best way to describe this - but it's not the guid or short id

Reply:
added search for it because test would find a pt with an onc history tab and fail later in onc history upload attempt because they didn't have a participantId i.e. didn't have an empty dsm.participant object according to the page error